### PR TITLE
Tooltip component adapted to the new design system

### PR DESCRIPTION
**Pull request by vmilan**
# Description

Shortcut: [#268623](https://app.shortcut.com/cartoteam/story/268623/components-basic-components-priority-2-1)
[sc-268623]

Tooltip component with the new design system applied.

Created a new component for building data structures within Tooltips, `TooltipData`.

### CODE REVIEW

File: UPGRADE.md [hunk 1]
```diff
@@ -104,3 +104,9 @@ So, instead of Mui Button, the component you should use to create buttons is thi
 `react-ui/src/atoms/Button`
 
 For external use: `import { Button } from '@carto/react-ui';`.
+
+## Tooltip
+
+Now, by default is placed `top` and has an `arrow` indicator, so you don't need to specify these properties anymore.
```
Comment (VictorVelarde) [inline UPGRADE.md:110]: ✨

File: packages/react-ui/src/components/organisms/TooltipData.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+      )}
+
+      <ul className={classes.list}>
+        {items.map((item, index) => (
+          <li key={index} className={classes.item}>
+            <span
+              className={classes.bullet}
+              style={{
+                backgroundColor: `${item.outlinedBullet ? 'transparent' : item.color}
+            `,
```
Comment (VictorVelarde) [inline packages/react-ui/src/components/organisms/TooltipData.js:55]: little formatting issue here?

[the remark this answers is not in the document]
Comment (vmilan) [reply packages/react-ui/src/components/organisms/TooltipData.js:55]: fixed, thanks!

Comment (vmilan) [reply packages/react-ui/src/components/organisms/TooltipData.js:55]: fixed, thanks!

File: packages/react-ui/src/components/organisms/TooltipData.js [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+      outlinedBullet: false
+    }
+  ]
+};
+
+TooltipData.propTypes = {
+  items: PropTypes.arrayOf(
+    PropTypes.shape({
+      category: PropTypes.string,
+      value: PropTypes.number.isRequired,
```
[the remark this answers is not in the document]
Comment (vmilan) [reply packages/react-ui/src/components/organisms/TooltipData.js:86]: The use case is a list with category and value text, which you can see here: https://www.figma.com/file/nmaoLeo69xBJCHm9nc6lEV/CARTO-Components-1.0?node-id=2080%3A30579&t=ydafko5KQ8kdV5t4-0

I think that react node is too broad, but I'll change it to a string or number

**Pull request merged**